connect/main.py

```diff
@@ -46,7 +46,10 @@ def get_app() -> FastAPI:
     # use the slowapi rate limiter
     app.add_middleware(SlowAPIMiddleware)
     limiter = Limiter(
-        key_func=get_remote_address, default_limits=[settings.connect_rate_limit]
+        key_func=get_remote_address,
+        auto_check=settings.connect_enable_rate_limit,
+        enabled=settings.connect_enable_rate_limit,
+        default_limits=[settings.connect_rate_limit],
     )
```

Comment:
It actually took some debugging to figure out that we need auto_check = True when enabled = True and auto_check = False when enabled = False.  Because the values of the 2 parameters will always be the same, we can transparently set auto_check to the value of enabled.

Reply:
Just posting for folks reading along
```
auto_check: whether to automatically check the rate limit in the
before_request chain of the application. default True
```

```
enabled: set to False to deactivate the limiter (default: True)
```
from https://slowapi.readthedocs.io/en/latest/api/

Reply:
Yes, there just aren't any examples of having to use auto_check and we very definitely do need it.  I posted a message under a related issue for slowapi and pointed out this alternate config.